### PR TITLE
Add missing links to the CompetenceCourse management interface ACDM-1361 #resolve

### PR DESCRIPTION
**Pull request by francisconeves97**

### CODE REVIEW

File: src/main/webapp/WEB-INF/fenixedu-academic/competence-courses/fragments/competenceCourseRow.jsp [hunk 1]
```diff
@@ -35,7 +35,21 @@
                 </c:when>
             </c:choose>
             <a class="btn btn-link btn-xs" href="${fr:checksumLink(pageContext.request, showCompetenceUrl)}"><spring:message code="label.show" text="Ver"/></a>
-        
+        	<c:if test="${isBolonhaManager && competenceCourse.curricularStage.name != 'APPROVED'}">
```
Comment (sergiofbsilva) [inline src/main/webapp/WEB-INF/fenixedu-academic/competence-courses/fragments/competenceCourseRow.jsp]: use {{competenceCourse .isApproved}} instead.

Comment (francisconeves97) [reply src/main/webapp/WEB-INF/fenixedu-academic/competence-courses/fragments/competenceCourseRow.jsp]: Made changes.

File: src/main/webapp/WEB-INF/fenixedu-academic/competence-courses/fragments/competenceCourseRow.jsp [hunk 2]
```diff
@@ -35,7 +35,21 @@
...
+				<spring:url var="deleteCompetenceUrl" context="" value="/bolonhaManager/competenceCourses/deleteCompetenceCourse.faces?action=ccm&competenceCourseID={competenceCourse}">
+					<spring:param name="competenceCourse" value="${competenceCourse.externalId}"/>
+				</spring:url>
+				<spring:url var="bibliographicReferenceUrl" context="" value="/bolonhaManager/competenceCourses/setCompetenceCourseBibliographicReference.faces?action=add&bibliographicReferenceID=-1&competenceCourseID={competenceCourse}">
+					<spring:param name="competenceCourse" value="${competenceCourse.externalId}"/>
+				</spring:url>
+				<a class="btn btn-link btn-xs" href="${fr:checksumLink(pageContext.request, editCompetenceUrl)}"><spring:message code="label.edit" text="Editar"/></a>
+				<a class="btn btn-link btn-xs" href="${fr:checksumLink(pageContext.request, deleteCompetenceUrl)}"><spring:message code="label.delete" text="Apagar"/></a>
+            	<a class="btn btn-link btn-xs" href="${fr:checksumLink(pageContext.request, bibliographicReferenceUrl)}"><spring:message code="label.bibliographicReference" text="Bibliografia"/></a>
+            </c:if>
```
Comment (sergiofbsilva) [inline src/main/webapp/WEB-INF/fenixedu-academic/competence-courses/fragments/competenceCourseRow.jsp]: It looks like the links are available only if the competenceCourse is approved which does not meet the issue requirement.

Comment (francisconeves97) [reply src/main/webapp/WEB-INF/fenixedu-academic/competence-courses/fragments/competenceCourseRow.jsp]: Changed the condition to only show links if competenceCourse is not approved (draft or published)

**Pull request merged**